ironfish-cli/src/commands/deposit.ts

```diff
@@ -0,0 +1,213 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import {
+  Assert,
+  displayIronAmountWithCurrency,
+  IronfishRpcClient,
+  ironToOre,
+  isValidAmount,
+  MINIMUM_IRON_AMOUNT,
+  oreToIron,
+  WebApi,
+} from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../command'
+import { RemoteFlags } from '../flags'
+import { ProgressBar } from '../types'
+
+const REGISTER_URL = 'https://testnet.ironfish.network/signup'
+const IRON_TO_SEND = 0.1
+
+export default class Bank extends IronfishCommand {
+  static description = 'Deposit $IRON for testnet points'
+
+  client: IronfishRpcClient | null = null
+  api: WebApi | null = new WebApi()
+
+  static flags = {
+    ...RemoteFlags,
+    fee: Flags.integer({
+      char: 'f',
+      default: 1,
+      description: `the fee amount in ORE, minimum of 1. 1 ORE is equal to ${MINIMUM_IRON_AMOUNT} IRON`,
+    }),
+    expirationSequence: Flags.integer({
+      char: 'e',
+      description: 'max number of blocks for the transaction to wait before expiring',
+    }),
+    account: Flags.string({
+      char: 'a',
+      default: 'default',
+      parse: (input) => Promise.resolve(input.trim()),
+      description: 'the account to send money from',
+    }),
+    confirm: Flags.boolean({
+      default: false,
+      description: 'confirm without asking',
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(Bank)
+    const fee = flags.fee
+    const feeInIron = oreToIron(fee)
+    const accountName = flags.account
+    const expirationSequence = flags.expirationSequence
+
+    this.client = await this.sdk.connectRpc()
+    this.api = new WebApi()
+
+    const bankDepositAddress = await this.api.getDepositAddress()
+    if (!bankDepositAddress) {
+      this.log('Error fetching deposit address. Please try again later.')
+      this.exit(1)
+    }
+
+    const { canSend, errorReason } = await this.verifyCanSend(flags)
+    if (!canSend) {
+      Assert.isNotNull(errorReason)
+      this.log(errorReason)
+      this.exit(1)
+    }
+
+    const graffiti = this.sdk.config.get('blockGraffiti')
```

Comment:
Should this come from the RPC layer so it'll match the node that sends the transaction, or no?